src/Pages/GradeSelect/index.js

```diff
@@ -1,7 +1,314 @@
+    setDropdownActive(!isDropdownActive);
   }
+
+  function toGoNextStage(e) {
+    e.preventDefault();
+    props.history.push('/gradeselect/arrival');
+  }
+
+  function handleSelected(target) {
+    const newSelectedDep = flightLists.filter((el) => el.id === target.id);
+    setSelectedDep(...newSelectedDep);
+  }
+
+  const price = +selectedDep.price;
+
+  return (
+    <div>
+      {console.log(selectedDep)}
+      <GradeSelectMain>
+        <Wrap>
+          <SearchWidget scrollTop={scrollTop} />
+          {/* num navi */}
+          <NumNavi>
+            <NaviLists>
+              <NaviList>
+                <NaviLink to="/main">
+                  <Numbering>1</Numbering>
+                  <NaviTitle>검색</NaviTitle>
+                </NaviLink>
+              </NaviList>
+              <NaviList>
+                <NaviLink to="/gradeselect">
+                  <Numbering className="active">2</Numbering>
+                  <NaviTitle className="focus">항공편</NaviTitle>
+                </NaviLink>
+              </NaviList>
+              <NaviList>
+                <NaviLink to="orderform">
+                  <Numbering className="undone">3</Numbering>
+                  <NaviTitle className="undone">결제</NaviTitle>
+                </NaviLink>
+              </NaviList>
+            </NaviLists>
+          </NumNavi>
+          {/* title*/}
+          <Title>
+            <TitleWrap>
+              <TitleHead> 가는 편 </TitleHead>
+              <p>
+                <TitlePara>GMP 서울/김포</TitlePara>
+                <TitlePara>→</TitlePara>
+                <TitlePara>CJU 제주</TitlePara>
+              </p>
+            </TitleWrap>
+          </Title>
+
+          <form>
+            <FlightList>
+              <FlightListWrap>
+                <FilterSelector
+                  toggleDropdown={toggleDropdown}
+                  isDropdownActive={isDropdownActive}
+                />
+                <ul>
+                  {!!flightLists.length &&
+                    flightLists.map((flightList) => {
+                      return (
+                        <FlightListCard
+                          key={flightList.id}
+                          list={flightList}
+                          handleSelected={handleSelected}
+                        />
+                      );
+                    })}
+                </ul>
+              </FlightListWrap>
+            </FlightList>
+            <SubBtnBar>
+              <SubBtnBarWrap>
+                <SubBtnBarTitle>예상 결제 금액</SubBtnBarTitle>
+                <SubBtnBarFare>
+                  <TotalFare>
+                    <FareNum>
+                      {!selectedDep.price ? 0 : price.toLocaleString()}
+                    </FareNum>
+                    원
+                  </TotalFare>
+                </SubBtnBarFare>
+                <PaymentBtn
+                  className={!!selectedDep.id && 'active'}
+                  onClick={(e) => toGoNextStage(e)}
```

Comment:
```suggestion
                  onClick={toGoNextStage}
```
가능한 경우에는 가급적 새로운 함수 인스턴스를 생성하지 않고 함수 자체를 넘기는 것이 좋습니다

Reply:
수정하였습니다!